### ISSUE TITLE
[Question] no command and variables after successful installation

### ISSUE THREAD

**Issue by poulpoulsen** [Phundrak/eshell-info-banner.el#15]
Hello,
it seems that something under the hood is going wrong.
I use emacs 28 on fedora.
installation from melpa runs without error.
but there is no command and no variables.
so i think, that something is wrong silently...
i tried it inside eshell buffer and outside.
Nothing :-(
any idea to start?

Regards
Poul

**Comment by Phundrak** [comment]
As per the installation instructions[1], you have to hook the function
`eshell-info-banner-update-banner` to the hook
`eshell-banner-load-hook`. Have you done that? I did not encounter any
issues on my end in my Fedora VM.

```lisp
(add-hook 'eshell-banner-load-hook #'eshell-info-banner-update-banner)
```
Or with `use-package`:
```lisp
(use-package eshell-info-banner
  :ensure t
  :defer t
  :hook (eshell-banner-load . eshell-info-banner-update-banner))
```



Footnotes
_________

[1] <https://github.com/Phundrak/eshell-info-banner.el/#installation>

--
Lucien “Phundrak” Cartier-Tilet

<https://phundrak.com> (Français)

<https://phundrak.com/en> (English)

Sent from GNU/Emacs

**Comment by poulpoulsen** [comment]
Hello,
thanks for the quick answer.
yes, i did it the same way.
(require 'eshell-info-banner)
(add-hook 'eshell-banner-load-hook 'eshell-info-banner-update-banner)

and unfortunately there is no error, no message.
this is strange.
can i set any variable or any special setting to view the start?
or any possibility to debug?

Regards
Poul

Sorry for my wording, i'm not a developer.

**Comment by Phundrak** [comment]
If you type `eshell-info-banner' in your eshell buffer, does it output
anything?

--
Lucien “Phundrak” Cartier-Tilet

<https://phundrak.com> (Français)

<https://phundrak.com/en> (English)

Sent from GNU/Emacs

**Comment by poulpoulsen** [comment]
Hello,
this is the message i get.

Wrong type argument: eshell-info-banner--mounted-partitions, "Filesystem      Size  Used Avail Use% Mounted on"

does it help?
Poul

**Comment by Phundrak** [comment]
This indeed sounds like an error, I guess with `df' and its output
processing, I’ll try to investigate it.

--
Lucien “Phundrak” Cartier-Tilet

<https://phundrak.com> (Français)

<https://phundrak.com/en> (English)

Sent from GNU/Emacs

**Comment by Phundrak** [comment]
Fixed with commit 0bd70d0d0f378f30d9effc49b9ff68baa3fa3066

--
Lucien “Phundrak” Cartier-Tilet

<https://phundrak.com> (Français)

<https://phundrak.com/en> (English)

Sent from GNU/Emacs
